test/functional/feature_bip68_sequence.py

```diff
@@ -214,19 +200,20 @@ def test_sequence_lock_confirmed_inputs(self):
                             sequence_value = ((cur_time - orig_time) >> SEQUENCE_LOCKTIME_GRANULARITY)+1
                         sequence_value |= SEQUENCE_LOCKTIME_TYPE_FLAG
                 tx.vin.append(CTxIn(COutPoint(int(utxos[j]["txid"], 16), utxos[j]["vout"]), nSequence=sequence_value))
-                value += utxos[j]["amount"]*COIN
+                value += utxos[j]["value"]*COIN
             # Overestimate the size of the tx - signatures should be less than 120 bytes, and leave 50 for the output
             tx_size = len(tx.serialize().hex())//2 + 120*num_inputs + 50
             tx.vout.append(CTxOut(int(value - self.relayfee * tx_size * COIN / 1000), SCRIPT_W0_SH_OP_TRUE))
-            rawtx = self.nodes[0].signrawtransactionwithwallet(tx.serialize().hex())["hex"]
+            rawtx = self.wallet.signrawtransactionwithwallet(tx=tx)["hex"]
 
             if (using_sequence_locks and not should_pass):
                 # This transaction should be rejected
-                assert_raises_rpc_error(-26, NOT_FINAL_ERROR, self.nodes[0].sendrawtransaction, rawtx)
+                assert_raises_rpc_error(-26, NOT_FINAL_ERROR, self.wallet.sendrawtransaction, from_node=self.nodes[0], tx_hex=rawtx)
             else:
                 # This raw transaction should be accepted
-                self.nodes[0].sendrawtransaction(rawtx)
-                utxos = self.nodes[0].listunspent()
+                self.wallet.sendrawtransaction(from_node=self.nodes[0], tx_hex=rawtx)
+                self.wallet.rescan_utxos()
```

Comment:
Why is this needed? `wallet.sendrawtransaction` should already account and scan the tx

Reply:
`utxos = self.wallet.get_utxos(include_immature_coinbase = False)` implies `mark_as_spent = True`, which means utxos is cleared.
`wallet.sendrawtransaction` only appends the tx to `self._utxos`, so we need `self.wallet.rescan_utxos()` to rescan unspent utxos.

Reply:
Ah, ok. So alternatively the `wallet.sendrawtransaction` could be dropped/reverted to just `sendrawtransaction`, because it is not needed to scan the tx.